src/synthesizer/particle/galaxy.py

```diff
@@ -206,6 +207,12 @@ def load_stars(
                 | (ages is None)
                 | (metallicities is None)
             ):
+                if verbose:
+                    print(
+                        ("In `load_stars`: one of either `initial_masses`"
```

Comment:
For the record these inner wrapping brackets are unnecessary because the print already wraps the strings.